lms/djangoapps/certificates/management/commands/cert_whitelist.py

```diff
@@ -12,6 +12,18 @@
 from django.contrib.auth.models import User
 
 
+def get_user(username):
+    """
+     This function takes the string username and fetch relavent user object from database
```

Comment:
looks like this function takes a username or email
maybe should be renamed to `get_user_from_identifier`